### DISCUSSION TITLE
Street Parking Lanes

### DISCUSSION THREAD

**Discussion by jeremyagulnek** [usdot-jpo-ode/jpo-wzdx#186] [Q&A] [answered]
Does the WZDx specification support street parking "lanes", i.e., portions of the roadway typically used for parking? Should this be supported as a Lane Type enumerated type?

**Comment by j-d-b** [comment, marked as answer]
@jeremyagulnek see [this comment in issue #149](https://github.com/usdot-jpo-ode/wzdx/issues/149#issuecomment-871033681). The addition of "parking" lanes is intended to be addressed in v4.0.

**Reply by jeremyagulnek** [reply]
Thank you!
